src/tests/ftest/util/dfuse_test_base.py

```diff
@@ -44,7 +44,7 @@ def stop_job_managers(self):
             error_list.append("Error stopping dfuse: {}".format(error))
         return error_list
 
-    def start_dfuse(self, hosts, pool=None, container=None, mount_dir=None):
+    def start_dfuse(self, hosts, pool=None, container=None, mount_dir=None, bind_cores=None):
```

Comment:
We should add a docstring line below for the new param

Reply:
I've done a big re-write and removed the option.